services/QuillLMS/app/models/provider_classroom_user.rb

```diff
@@ -9,24 +9,18 @@
 #  type                  :string           not null
 #  created_at            :datetime         not null
 #  updated_at            :datetime         not null
-#  canvas_instance_id    :bigint
 #  classroom_external_id :string           not null
 #  user_external_id      :string           not null
 #
 # Indexes
 #
-#  index_provider_classroom_users_on_canvas_instance_id  (canvas_instance_id)
-#  index_provider_type_and_classroom_id_and_user_id      (type,classroom_external_id,user_external_id) UNIQUE
-#
-# Foreign Keys
-#
-#  fk_rails_...  (canvas_instance_id => canvas_instances.id)
+#  index_provider_type_and_classroom_id_and_user_id  (type,classroom_external_id,user_external_id) UNIQUE
 #
 class ProviderClassroomUser < ApplicationRecord
   ACTIVE = :active
   DELETED = :deleted
 
-  TYPES = %w[CleverClassroomUser GoogleClassroomUser].freeze
+  TYPES = %w[CanvasClassroomUser CleverClassroomUser GoogleClassroomUser].freeze
```

Comment:
The delimiters are stylistic, you can use different ones:
  
Square brackets: %w[...]
Curly braces: %w{...}
Parentheses: %w(...)
Less than/greater than: %w<...>

I tend to prefer square brackets since the return type is an array.